checkov/main.py

```diff
@@ -211,7 +211,8 @@ def run(banner: str = checkov_banner, argv: list[str] = sys.argv[1:]) -> int | N
                                  block_list_secret_scan=config.block_list_secret_scan,
                                  deep_analysis=config.deep_analysis,
                                  repo_root_for_plan_enrichment=config.repo_root_for_plan_enrichment,
-                                 enable_git_history_secret_scan=config.scan_secrets_history)
+                                 enable_git_history_secret_scan=config.scan_secrets_history,
+                                 git_history_timeout=config.secrets_history_timeout)
```

Comment:
```suggestion
                                 enable_git_history_timeout=config.secrets_history_timeout)
```

Reply:
that's the actual timeout.. not a boolean. so enable doesn't make sense to me here